api/api_ethereum.go

```diff
@@ -1132,77 +1213,127 @@ func (args *EthTransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int,
 	data := args.data()
 
 	// TODO-Klaytn: Klaytn does not support accessList yet.
-	// var accessList AccessList
+	// var accessList types.AccessList
 	// if args.AccessList != nil {
 	//	 accessList = *args.AccessList
 	// }
 	return types.NewMessage(addr, args.To, 0, value, gas, gasPrice, data, false, intrinsicGas), nil
 }
 
+// toTransaction converts the arguments to a transaction.
+// This assumes that setDefaults has been called.
+func (args *EthTransactionArgs) toTransaction() *types.Transaction {
+	var tx *types.Transaction
+	switch {
+	case args.MaxFeePerGas != nil:
```

Comment:
We not support `MaxPriorityFeePerGas != nil` case?

Reply:
Yeap. It is ok to consider one field `MaxFeePerGas` or `MaxPriorityFeePerGas`. (Ethereum takes MaxFeePerGas for checking and I applied it too.)

It because `MaxFeePerGas` and `MaxPriorityFeePerGas` always come together. You can check that all relevant fields are set before this function is used.